readme.md

```diff
@@ -443,6 +443,10 @@ done manually or using OpenTracing framework instrumentations such as
 Framework instrumentation automatically propagates tracing headers and also creates in-process
 spans to better understand what is happening inside the application.
 
+There are different ways how the tracer can be configured. Customer Java service is using `tracerresolver`
+which does not require any code changes and the whole configuration is moved to environmental properties. 
+Whereas preference service is instantiating tracer bean directly in Spring configuration class.
```

Comment:
Slight change: "Whereas the Preference Java service is instantiating the tracer...."